docs/src/content/docs/it/guides/css-and-tailwind.mdx

```diff
@@ -0,0 +1,289 @@
+
+   ```sh
+   yarn astro add tailwind
+   ```
+
+   </TabItem>
+
+   </Tabs>
+
+2. Installa il plugin Starlight Tailwind:
+
+   <Tabs>
+
+   <TabItem label="npm">
+
+   ```sh
+   npm install @astrojs/starlight-tailwind
+   ```
+
+   </TabItem>
+
+   <TabItem label="pnpm">
+
+   ```sh
+   pnpm install @astrojs/starlight-tailwind
+   ```
+
+   </TabItem>
+
+   <TabItem label="Yarn">
+
+   ```sh
+   yarn add @astrojs/starlight-tailwind
+   ```
+
+   </TabItem>
+
+   </Tabs>
+
+3. Crea un file CSS per gli stili di base di Tailwind, ad esempio in `src/tailwind.css`:
+
+   ```css
+   /* src/tailwind.css */
+   @tailwind base;
+   @tailwind components;
+   @tailwind utilities;
+   ```
+
+4. Aggiorna il file di configurazione Astro per utilizzare gli stili di base Tailwind e disabilitare gli stili di base predefiniti:
+
+   ```js {11-12,16-17}
+   // astro.config.mjs
+   import { defineConfig } from 'astro/config';
+   import starlight from '@astrojs/starlight';
+   import tailwind from '@astrojs/tailwind';
+
+   export default defineConfig({
+   	integrations: [
+   		starlight({
+   			title: 'Docs with Tailwind',
+   			customCss: [
+   				// Percorso per gli stili di base di Tailwind
+   				'./src/tailwind.css',
+   			],
+   		}),
+   		tailwind({
+   			// Disattiva gli stili di base predefiniti
+   			applyBaseStyles: false,
+   		}),
+   	],
+   });
+   ```
+
+5. Aggiungi il plugin Starlight Tailwind a `tailwind.config.cjs`:
+
+   ```js ins={2,7}
+   // tailwind.config.cjs
+   const starlightPlugin = require('@astrojs/starlight-tailwind');
+
+   /** @type {import('tailwindcss').Config} */
+   module.exports = {
+   	content: ['./src/**/*.{astro,html,js,jsx,md,mdx,svelte,ts,tsx,vue}'],
+   	plugins: [starlightPlugin()],
+   };
+   ```
+
+### Styling di Starlight con Tailwind
+
+Starlight utilizzerà i valori della [configurazione del tema Tailwind](https://tailwindcss.com/docs/theme) nella sua interfaccia utente.
+
+Se impostate, le seguenti opzioni sovrascriveranno gli stili predefiniti di Starlight:
+
+- `colors.accent` — utilizzato per i collegamenti e per l'evidenziazione dell'elemento corrente
+- `colors.gray`: utilizzato per i colori e i bordi dello sfondo
+- `fontFamily.sans`: utilizzato per l'interfaccia utente e il testo del contenuto
+- `fontFamily.mono` — utilizzato per esempi di codice
+
+```js {12,14,18,20}
+// tailwind.config.cjs
+const starlightPlugin = require('@astrojs/starlight-tailwind');
+const colors = require('tailwindcss/colors');
+
+/** @type {import('tailwindcss').Config} */
+module.exports = {
+	content: ['./src/**/*.{astro,html,js,jsx,md,mdx,svelte,ts,tsx,vue}'],
+	theme: {
+		extend: {
+			colors: {
+				// Il tuo colore preferito. Indigo è il più vicino alle impostazioni predefinite di Starlight.
+				accent: colors.indigo,
+				// La tua scala di grigi preferita. Lo zinco è il più vicino ai valori predefiniti di Starlight.
+				gray: colors.zinc,
+			},
+			fontFamily: {
+				sans: ['"Atkinson Iperleggibile"'],
```

Comment:
Questo è un vero font — https://brailleinstitute.org/freefont — quindi probabilmente meglio non tradurre il nome:
```suggestion
				sans: ['"Atkinson Hyperlegible"'],
```

Reply:
Questo è ciò che succede quando lo fai la sera tardi, partendo da una traduzione in parte automatica...